ibis/backends/dask/tests/execution/test_operations.py

```diff
@@ -427,9 +427,7 @@ def test_nullif_inf(npartitions):
     expected = dd.from_pandas(
         pd.Series([np.nan, 3.14, np.nan, 42.0], name="a"),
         npartitions=npartitions,
-    ).reset_index(
-        drop=True
-    )  # match dask reset index behavior
+    ).reset_index(drop=True)  # match dask reset index behavior
```

Comment:
This looks like an improvement over Black. Again, the following comment must have confused it.

Reply:
The line is only 63 characters long,  so I don't think this should be the case.